modules/mobile/spec/request/v1/messages_request_spec.rb

```diff
@@ -17,22 +18,92 @@
     allow_any_instance_of(MHVAccountTypeService).to receive(:mhv_account_type).and_return(mhv_account_type)
     allow(Mobile::V0::Messaging::Client).to receive(:new).and_return(authenticated_client)
     iam_sign_in(build(:iam_user, iam_mhv_id: '123'))
+    @original_cassette_dir = VCR.configure(&:cassette_library_dir)
```

Comment:
You also need to reset this in an after block. There are examples throughout our test suite. This is why I wish our cassettes were in the root level like everyone else's. I suspect there is or was once a restriction on the root level VCR folder and you needed outside approval to merge, which is why we created our own directory. I haven't tested or looked into this, but I suspect that's probably no longer true and we could probably use the shared directory now. 

The alternative would be to create our own rails helper that does this for our entire suite. I tried doing it once but ran into some load path issues and decided it wasn't worth the effort.